requirements.txt

```diff
@@ -8,3 +8,4 @@ matplotlib>=3.3
 galsim>=2.3
 treegp>=0.6
 threadpoolctl>=3.1
+jax>=0.4
```

Comment:
As mentioned above, I think we can probably remove this from the requirements and make it optional.